### ISSUE TITLE
[BUG] 5.1.22 upgrade places `$:/UpgradeWizard` `$:/tags/AboveStory`

### ISSUE THREAD

**Issue by pinysuse** [Jermolene/TiddlyWiki5#4879]
# Issue

5.1.22 upgrade places `$:/UpgradeWizard` `$:/tags/AboveStory` which is annoying

## Steps to reproduce the behavior:
1. Download [TiddlyWiki Upgrade Wizard 5.1.22](https://tiddlywiki.com/upgrade.html)
1. Follow instructions and save upgraded wiki
1. Open saved wiki
1. On top of wiki the Upgrade Wizard message appears

## Expected behavior
No Upgrade Wizard message should cover the screen

## Desktop
  
OS:     Linux 4.12.14-lp151.28.67-default x86_64
System: openSUSE Leap 15.1
Qt:     5.9.7
KDE:    5.55.0
Plasma: 5.12.8
FF: 81.0

**Comment by pinysuse** [comment]
# Mitigation

Disable plugin `$:/plugins/tiddlywiki/upgrade` on the `$:/ControlPanel`

**Comment by Jermolene** [comment]
Thanks @pinysuse I was unable to reproduce the problem using Firefox on macOS. One possibility is that the wiki you are trying to upgrade contains system tiddlers that interfere with the upgrade process, but it's not obvious how that could happen. Another thing to note is that using the browsers own File->Save As menu item doesn't work; one needs to click the big green "Save upgraded TiddlyWiki file" button instead.

**Comment by pinysuse** [comment]
Thanks @Jermolene for the hint
> One possibility is that the wiki you are trying to upgrade contains system tiddlers that interfere with the upgrade process (...)

Tested with some old wikis from my backup and I do believe that plugins to not interfere here in general.

> Another thing to note is that using the browsers own File->Save As menu item doesn't work; one needs to click the big green "Save upgraded TiddlyWiki file" button instead.

However, I am using AddOn [File Backup Utility For TiddlyWiki]( https://pmario.github.io/file-backups/) which automatically saves the new wiki upon hitting **_upgrade_**. 
This leaves a big upgrade.html in the ~/Downloads folder (19.6MiB) which still contains the upgrade plugin code.
When additionally hitting the **_big green button_** a smaller file is created (6MiB) w/o that upgrade plugin present - as intended!

- [X] Cross check w/ diabled  [File Backup Utility For TiddlyWiki] → no auto save → **_big green button_** → OK!

So the problem was once again in front of the screen - thanks for your advice!

4YI @pmario just in case...

**Comment by pmario** [comment]
I'll test this. Thx for your tests and digging out the problem.

**Comment by pmario** [comment]
I did  a test with the online version and did upgrade TW V5.1.0 (the very first version). Everything works as expected. 
Next test will be with an upgrade version started from a file:/// URI

**Comment by pmario** [comment]
> - Apparently I did not follow instructions but grabbed that autosave half baked wiki!

That's right. File-Backups did save the upgrade.html .. Which also contains the stuff needed to upgrade. ... 

IMO the wording in the instructions may be changed a little bit. So it makes it clear that the "green button" has to be clicked, even if the update.html file is saved by an "autosave" event. ... 

It may be possible to disable the "autosave" setting in the update.html file. 

Second: The green button should save the wiki named: `<old-wiki-name>-v5.1.22.html` or something similar. For me it is confusing, that it uses upgrade.html

**Comment by pmario** [comment]
@Jermolene ... What do you think of disabling the "autosave" option for the "upgrade.html", if it is used from a file:// uri. 

IMO there would be 2 advantages. 

 - The upgrade.html may be used for several wikis
 - There is no automatic "wiki saved" message, if browser addOns are installed. 

And we ma change the name of the "upgraded" file.

**Comment by Jermolene** [comment]
Hi @pmario and apologies for the delay

> What do you think of disabling the "autosave" option for the "upgrade.html", if it is used from a file:// uri.

Wouldn't it make sense to disable autosave regardless of the URL? It will only be enabled if the wiki being upgraded has enabled it. I'll commit a quick fix shortly, let me know if it works for you.

**Comment by saqimtiaz** [comment]
@Jermolene you resolved this a few weeks ago, can be closed.

**Comment by pinysuse** [comment]
Thanks again @Jermolene and @pmario for taking care!
